ingestion/functions/parsing/peru/peru.py

```diff
@@ -72,37 +107,43 @@ def parse_cases(raw_data_file, source_id, source_url):
     "Lima" is often provided as all three locations ("Lima, Lima, Lima, Peru") - to geocode this with mapbox
```

Comment:
Notice you no longer do this so should probably remove this comment.

Reply:
Agreed